hedgehog/src/Hedgehog/Internal/State.hs

```diff
@@ -35,7 +35,6 @@ module Hedgehog.Internal.State (
   -- * Commands
   , Command(..)
   , Callback(..)
-  , commandGenOK
```

Comment:
Note to self: Remember to bump the version appropriately for this.

Reply:
See https://github.com/hedgehogqa/haskell-hedgehog/pull/554#issuecomment-3287624883.